### PR TITLE
linux-mint: move to dockerscripts/linuxmint image

### PR DESCRIPTION
**Pull request by CAMOBAP**

### CODE REVIEW

**Comment by CAMOBAP** [comment]
Looks like wayback.machine.org is down, will rerun after some time

**Comment by CAMOBAP** [comment]
https://mastodon.archive.org/@internetarchive/112513905401989149 it's under DDoS and still fown

File: .github/workflows/linux-mint.yml [hunk 1]
```diff
@@ -9,9 +9,9 @@ on:
 jobs:
   build_without_convert:
     runs-on: ubuntu-latest
-    container: vcatechnology/linux-mint
+    container: dockerscripts/linuxmint
```
Comment (CAMOBAP) [inline .github/workflows/linux-mint.yml:12]:
```suggestion
    container: dockerscripts/linuxmint:xfce-minimal
```

File: .github/workflows/linux-mint.yml [hunk 2]
```diff
@@ -45,9 +45,9 @@ jobs:
 
   build_and_except_eula_manually:
     runs-on: ubuntu-latest
-    container: vcatechnology/linux-mint
+    container: dockerscripts/linuxmint
```
Comment (CAMOBAP) [inline .github/workflows/linux-mint.yml:48]:
```suggestion
    container: dockerscripts/linuxmint:xfce-minimal
```

File: .github/workflows/linux-mint.yml [hunk 3]
```diff
@@ -62,9 +62,9 @@ jobs:
 
   build_and_reject_eula_manually:
     runs-on: ubuntu-latest
-    container: vcatechnology/linux-mint
+    container: dockerscripts/linuxmint
```
Comment (CAMOBAP) [inline .github/workflows/linux-mint.yml:65]:
```suggestion
    container: dockerscripts/linuxmint:xfce-minimal
```

**Comment by CAMOBAP** [comment]
@HassanAkbar ping

**Comment by ronaldtse** [comment]
@CAMOBAP we want to deprecate and archive the script, so let's not update the script anymore. 

Instead, can you help write a block post on how to use Fontist for people who want to use the script? Then we can duplicate the script and replace the README with a link to that blog post. Thanks!

**Comment by CAMOBAP** [comment]
@ronaldtse why not archive all repos that are deprecated? This will help a lot to understand when something isn't supported anymore

**Review by HassanAkbar — APPROVED** [review]

**Pull request merged**